src/OpenTelemetry.Api.ProviderBuilderExtensions/Logs/OpenTelemetryDependencyInjectionLoggerProviderBuilderExtensions.cs

```diff
@@ -26,8 +26,26 @@ namespace OpenTelemetry.Logs;
 /// <summary>
 /// Contains extension methods for the <see cref="LoggerProviderBuilder"/> class.
 /// </summary>
-public static class OpenTelemetryDependencyInjectionLoggerProviderBuilderExtensions
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    public
+#else
+    internal
+#endif
+    static class OpenTelemetryDependencyInjectionLoggerProviderBuilderExtensions
 {
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    /// <summary>
+    /// Adds instrumentation to the provider.
+    /// </summary>
+    /// <remarks>
+    /// <para><inheritdoc cref="AddInstrumentation{T}(LoggerProviderBuilder, T)" path="/remarks"/></para>
+    /// Note: The type specified by <typeparamref name="T"/> will be
```

Comment:
Add the warning for this method as well.

Reply:
It is there on line 41 via the `inheritdoc`. Easy to miss 😄 I did my best to not copy the warning text everywhere. It should be at most in 1 spot per assembly. In SDK there are 2, one for logs and one for exemplars. Because I figured it was a good idea to detach those two conceptual areas from each other.